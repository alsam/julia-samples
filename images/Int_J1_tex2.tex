\documentclass{minimal}
\usepackage{pgfplots}
\usepackage{fontspec}
\usepackage{amsmath}
\usepackage[active,tightpage]{preview}
\PreviewEnvironment{tikzpicture}
\begin{document}
\begin{tikzpicture}[x=1mm,y=-1mm]
\definecolor{mycolor000000}{rgb}{0,0,0}
\definecolor{mycolor000000}{rgb}{0,0,0}
\definecolor{mycolorD0D0E0}{rgb}{0.82,0.82,0.88}
\definecolor{mycolor00BFFF}{rgb}{0,0.75,1}
\definecolor{mycolor564A55}{rgb}{0.34,0.29,0.33}
\definecolor{mycolor6C606B}{rgb}{0.42,0.38,0.42}
\begin{scope}
\begin{scope}
\fontspec{PT Sans}
\draw (70.32,64.59) node [text=mycolor564A55,draw=mycolor000000,draw opacity=0,rotate around={-0: (0,1.81)},inner sep=0.0]{\fontsize{3.88mm}{4.66mm}\selectfont $\text{r}$};
\end{scope}
\begin{scope}
\fontspec{PT Sans Caption}
\draw (20.63,57.91) node [text=mycolor6C606B,rotate around={-0: (49.68,1.34)},inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{0}$};
\draw (70.32,57.91) node [text=mycolor6C606B,rotate around={-0: (0,1.34)},inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{50}$};
\draw (120,57.91) node [text=mycolor6C606B,rotate around={-0: (-49.68,1.34)},inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{100}$};
\end{scope}
\begin{scope}
\clip  (18.63,5) -- (122,5) -- (122,56.91) -- (18.63,56.91);
\begin{scope}
\clip  (18.63,5) -- (122,5) -- (122,56.91) -- (18.63,56.91);
\path [fill=mycolor000000,fill opacity=0,draw=mycolor000000,draw opacity=0] (18.63,5) rectangle +(103.37,51.91);
\end{scope}
\begin{scope}
[dash pattern=on 0.5mm off 0.5mm,line width=0.2mm]
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (18.63,54.91) -- (122,54.91);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (18.63,38.94) -- (122,38.94);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (18.63,22.97) -- (122,22.97);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (18.63,7) -- (122,7);
\end{scope}
\begin{scope}
[dash pattern=on 0.5mm off 0.5mm,line width=0.2mm]
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (20.63,5) -- (20.63,56.91);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (70.32,5) -- (70.32,56.91);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (120,5) -- (120,56.91);
\end{scope}
\begin{scope}
\begin{scope}
[line width=0.3mm]
\path [fill=mycolor000000,fill opacity=0,draw=mycolor00BFFF]  (20.63,54.91) -- (21.03,53.64) -- (21.43,49.97) -- (21.83,44.33) -- (22.23,37.4) -- (22.63,29.98) -- (23.03,22.89) -- (23.43,16.91) -- (23.82,12.64) -- (24.22,10.41) -- (24.62,10.32) -- (25.02,12.15) -- (25.42,15.48) -- (25.82,19.68) -- (26.22,24.07) -- (26.62,27.99) -- (27.02,30.89) -- (27.42,32.39) -- (27.81,32.34) -- (28.21,30.85) -- (28.61,28.21) -- (29.01,24.89) -- (29.41,21.42) -- (29.81,18.35) -- (30.21,16.13) -- (30.61,15.07) -- (31.01,15.28) -- (31.41,16.68) -- (31.81,19.02) -- (32.2,21.89) -- (32.6,24.84) -- (33,27.39) -- (33.4,29.17) -- (33.8,29.93) -- (34.2,29.57) -- (34.6,28.19) -- (35,26.03) -- (35.4,23.44) -- (35.8,20.86) -- (36.2,18.67) -- (36.59,17.21) -- (36.99,16.7) -- (37.39,17.18) -- (37.79,18.56) -- (38.19,20.61) -- (38.59,22.98) -- (38.99,25.3) -- (39.39,27.2) -- (39.79,28.39) -- (40.19,28.71) -- (40.59,28.12) -- (40.98,26.73) -- (41.38,24.77) -- (41.78,22.57) -- (42.18,20.47) -- (42.58,18.8) -- (42.98,17.83) -- (43.38,17.68) -- (43.78,18.37) -- (44.18,19.77) -- (44.58,21.66) -- (44.97,23.72) -- (45.37,25.63) -- (45.77,27.09) -- (46.17,27.88) -- (46.57,27.88) -- (46.97,27.1) -- (47.37,25.69) -- (47.77,23.87) -- (48.17,21.93) -- (48.57,20.18) -- (48.97,18.9) -- (49.36,18.28) -- (49.76,18.42) -- (50.16,19.26) -- (50.56,20.69) -- (50.96,22.45) -- (51.36,24.28) -- (51.76,25.87) -- (52.16,26.99) -- (52.56,27.45) -- (52.96,27.21) -- (53.36,26.29) -- (53.75,24.86) -- (54.15,23.15) -- (54.55,21.43) -- (54.95,19.98) -- (55.35,19.01) -- (55.75,18.68) -- (56.15,19.04) -- (56.55,20.01) -- (56.95,21.44) -- (57.35,23.1) -- (57.75,24.72) -- (58.14,26.04) -- (58.54,26.87) -- (58.94,27.07) -- (59.34,26.62) -- (59.74,25.59) -- (60.14,24.16) -- (60.54,22.56) -- (60.94,21.04) -- (61.34,19.84) -- (61.74,19.15) -- (62.13,19.07) -- (62.53,19.6) -- (62.93,20.67) -- (63.33,22.09) -- (63.73,23.64) -- (64.13,25.07) -- (64.53,26.15) -- (64.93,26.72) -- (65.33,26.69) -- (65.73,26.07) -- (66.13,24.97) -- (66.52,23.56) -- (66.92,22.07) -- (67.32,20.73) -- (67.72,19.76) -- (68.12,19.31) -- (68.52,19.44) -- (68.92,20.13) -- (69.32,21.27) -- (69.72,22.66) -- (70.12,24.1) -- (70.52,25.34) -- (70.91,26.2) -- (71.31,26.54) -- (71.71,26.31) -- (72.11,25.55) -- (72.51,24.39) -- (72.91,23.02) -- (73.31,21.65) -- (73.71,20.5) -- (74.11,19.74) -- (74.51,19.51) -- (74.91,19.83) -- (75.3,20.64) -- (75.7,21.82) -- (76.1,23.17) -- (76.5,24.48) -- (76.9,25.54) -- (77.3,26.19) -- (77.7,26.32) -- (78.1,25.92) -- (78.5,25.06) -- (78.9,23.87) -- (79.29,22.54) -- (79.69,21.3) -- (80.09,20.33) -- (80.49,19.78) -- (80.89,19.74) -- (81.29,20.22) -- (81.69,21.13) -- (82.09,22.32) -- (82.49,23.62) -- (82.89,24.8) -- (83.29,25.68) -- (83.68,26.13) -- (84.08,26.07) -- (84.48,25.53) -- (84.88,24.58) -- (85.28,23.38) -- (85.68,22.13) -- (86.08,21.01) -- (86.48,20.22) -- (86.88,19.86) -- (87.28,20.01) -- (87.68,20.62) -- (88.07,21.6) -- (88.47,22.79) -- (88.87,24.01) -- (89.27,25.05) -- (89.67,25.76) -- (90.07,26.02) -- (90.47,25.8) -- (90.87,25.13) -- (91.27,24.12) -- (91.67,22.93) -- (92.07,21.76) -- (92.46,20.79) -- (92.86,20.17) -- (93.26,19.99) -- (93.66,20.29) -- (94.06,21.02) -- (94.46,22.05) -- (94.86,23.22) -- (95.26,24.35) -- (95.66,25.25) -- (96.06,25.78) -- (96.45,25.87) -- (96.85,25.49) -- (97.25,24.72) -- (97.65,23.68) -- (98.05,22.52) -- (98.45,21.45) -- (98.85,20.62) -- (99.25,20.17) -- (99.65,20.17) -- (100.05,20.61) -- (100.45,21.42) -- (100.84,22.48) -- (101.24,23.61) -- (101.64,24.63) -- (102.04,25.39) -- (102.44,25.75) -- (102.84,25.68) -- (103.24,25.17) -- (103.64,24.32) -- (104.04,23.26) -- (104.44,22.16) -- (104.84,21.19) -- (105.23,20.51) -- (105.63,20.22) -- (106.03,20.37) -- (106.43,20.94) -- (106.83,21.82) -- (107.23,22.88) -- (107.63,23.96) -- (108.03,24.87) -- (108.43,25.47) -- (108.83,25.68) -- (109.23,25.45) -- (109.62,24.83) -- (110.02,23.92) -- (110.42,22.87) -- (110.82,21.83) -- (111.22,20.98) -- (111.62,20.45) -- (112.02,20.32) -- (112.42,20.62) -- (112.82,21.28) -- (113.22,22.21) -- (113.61,23.26) -- (114.01,24.26) -- (114.41,25.05) -- (114.81,25.5) -- (115.21,25.56) -- (115.61,25.2) -- (116.01,24.49) -- (116.41,23.54) -- (116.81,22.5) -- (117.21,21.55) -- (117.61,20.82) -- (118,20.44) -- (118.4,20.46) -- (118.8,20.88) -- (119.2,21.63) -- (119.6,22.59) -- (120,23.61);
\end{scope}
\end{scope}
\end{scope}
\begin{scope}
\fontspec{PT Sans Caption}
\draw (17.63,54.91) node [text=mycolor6C606B,rotate around={-0: (-2.01,-23.96)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{0.0}$};
\draw (17.63,38.94) node [text=mycolor6C606B,rotate around={-0: (-2.01,-7.99)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{0.5}$};
\draw (17.63,22.97) node [text=mycolor6C606B,rotate around={-0: (-2.01,7.99)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{1.0}$};
\draw (17.63,7) node [text=mycolor6C606B,rotate around={-0: (-2.01,23.96)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{1.5}$};
\end{scope}
\begin{scope}
\fontspec{PT Sans}
\draw (8.81,28.96) node [text=mycolor564A55,draw=mycolor000000,draw opacity=0,rotate around={90: (0,2)},inner sep=0.0]{\fontsize{3.88mm}{4.66mm}\selectfont $\text{$\int_{z=0}^{z=\textbf{r}}J_1(z)dz$}$};
\end{scope}
\end{scope}
\end{tikzpicture}
\end{document}
