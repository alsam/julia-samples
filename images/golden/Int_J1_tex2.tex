\documentclass{minimal}
\usepackage{pgfplots}
\usepackage{fontspec}
\usepackage{amsmath}
\usepackage[active,tightpage]{preview}
\PreviewEnvironment{tikzpicture}
\begin{document}
\begin{tikzpicture}[x=1mm,y=-1mm]
\definecolor{mycolor00BFFF}{rgb}{0,0.75,1}
\definecolor{mycolor564A55}{rgb}{0.34,0.29,0.33}
\definecolor{mycolor000000}{rgb}{0,0,0}
\definecolor{mycolor000000}{rgb}{0,0,0}
\definecolor{mycolor6C606B}{rgb}{0.42,0.38,0.42}
\definecolor{mycolorD0D0E0}{rgb}{0.82,0.82,0.88}
\begin{scope}
\begin{scope}
\fontspec{PT Sans}
\draw (70.91,66.08) node [text=mycolor564A55,draw=mycolor000000,draw opacity=0,rotate around={-0: (0,1.06)},inner sep=0.0]{\fontsize{3.88mm}{4.66mm}\selectfont $\text{r}$};
\end{scope}
\begin{scope}
\fontspec{PT Sans Caption}
\draw (21.82,60.32) node [text=mycolor6C606B,rotate around={-0: (49.09,0.88)},inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{0}$};
\draw (70.91,60.32) node [text=mycolor6C606B,rotate around={-0: (0,0.88)},inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{50}$};
\draw (120,60.32) node [text=mycolor6C606B,rotate around={-0: (-49.09,0.88)},inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{100}$};
\end{scope}
\begin{scope}
\clip  (19.82,5) -- (122,5) -- (122,59.32) -- (19.82,59.32);
\begin{scope}
\clip  (19.82,5) -- (122,5) -- (122,59.32) -- (19.82,59.32);
\path [fill=mycolor000000,fill opacity=0,draw=mycolor000000,draw opacity=0] (19.82,5) rectangle +(102.18,54.32);
\end{scope}
\begin{scope}
[dash pattern=on 0.5mm off 0.5mm,line width=0.2mm]
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (19.82,57.32) -- (122,57.32);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (19.82,40.55) -- (122,40.55);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (19.82,23.77) -- (122,23.77);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (19.82,7) -- (122,7);
\end{scope}
\begin{scope}
[dash pattern=on 0.5mm off 0.5mm,line width=0.2mm]
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (21.82,5) -- (21.82,59.32);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (70.91,5) -- (70.91,59.32);
\path [fill=mycolor000000,draw=mycolorD0D0E0]  (120,5) -- (120,59.32);
\end{scope}
\begin{scope}
\begin{scope}
[line width=0.3mm]
\path [fill=mycolor000000,fill opacity=0,draw=mycolor00BFFF]  (21.82,57.32) -- (22.21,55.98) -- (22.61,52.12) -- (23,46.21) -- (23.4,38.93) -- (23.79,31.13) -- (24.19,23.69) -- (24.58,17.41) -- (24.97,12.92) -- (25.37,10.59) -- (25.76,10.49) -- (26.16,12.41) -- (26.55,15.9) -- (26.95,20.31) -- (27.34,24.93) -- (27.73,29.05) -- (28.13,32.09) -- (28.52,33.66) -- (28.92,33.61) -- (29.31,32.05) -- (29.71,29.28) -- (30.1,25.79) -- (30.49,22.15) -- (30.89,18.92) -- (31.28,16.59) -- (31.68,15.47) -- (32.07,15.69) -- (32.47,17.17) -- (32.86,19.62) -- (33.25,22.64) -- (33.65,25.73) -- (34.04,28.42) -- (34.44,30.29) -- (34.83,31.08) -- (35.23,30.7) -- (35.62,29.25) -- (36.01,26.98) -- (36.41,24.27) -- (36.8,21.55) -- (37.2,19.25) -- (37.59,17.72) -- (37.99,17.18) -- (38.38,17.69) -- (38.77,19.14) -- (39.17,21.29) -- (39.56,23.78) -- (39.96,26.21) -- (40.35,28.21) -- (40.75,29.47) -- (41.14,29.8) -- (41.53,29.18) -- (41.93,27.72) -- (42.32,25.66) -- (42.72,23.35) -- (43.11,21.14) -- (43.51,19.4) -- (43.9,18.37) -- (44.29,18.21) -- (44.69,18.94) -- (45.08,20.41) -- (45.48,22.39) -- (45.87,24.56) -- (46.27,26.56) -- (46.66,28.1) -- (47.05,28.92) -- (47.45,28.93) -- (47.84,28.11) -- (48.24,26.63) -- (48.63,24.71) -- (49.03,22.68) -- (49.42,20.84) -- (49.81,19.5) -- (50.21,18.85) -- (50.6,18.99) -- (51,19.88) -- (51.39,21.37) -- (51.79,23.23) -- (52.18,25.15) -- (52.57,26.82) -- (52.97,27.99) -- (53.36,28.48) -- (53.76,28.22) -- (54.15,27.26) -- (54.55,25.76) -- (54.94,23.96) -- (55.33,22.16) -- (55.73,20.63) -- (56.12,19.61) -- (56.52,19.27) -- (56.91,19.64) -- (57.31,20.67) -- (57.7,22.17) -- (58.09,23.91) -- (58.49,25.61) -- (58.88,27) -- (59.28,27.87) -- (59.67,28.07) -- (60.07,27.6) -- (60.46,26.53) -- (60.86,25.03) -- (61.25,23.34) -- (61.64,21.74) -- (62.04,20.48) -- (62.43,19.76) -- (62.83,19.67) -- (63.22,20.24) -- (63.62,21.36) -- (64.01,22.85) -- (64.4,24.48) -- (64.8,25.97) -- (65.19,27.11) -- (65.59,27.71) -- (65.98,27.68) -- (66.38,27.03) -- (66.77,25.87) -- (67.16,24.39) -- (67.56,22.82) -- (67.95,21.42) -- (68.35,20.4) -- (68.74,19.93) -- (69.14,20.07) -- (69.53,20.79) -- (69.92,21.98) -- (70.32,23.45) -- (70.71,24.95) -- (71.11,26.26) -- (71.5,27.16) -- (71.9,27.52) -- (72.29,27.28) -- (72.68,26.48) -- (73.08,25.27) -- (73.47,23.82) -- (73.87,22.38) -- (74.26,21.17) -- (74.66,20.38) -- (75.05,20.14) -- (75.44,20.47) -- (75.84,21.33) -- (76.23,22.56) -- (76.63,23.98) -- (77.02,25.36) -- (77.42,26.47) -- (77.81,27.15) -- (78.2,27.29) -- (78.6,26.87) -- (78.99,25.96) -- (79.39,24.71) -- (79.78,23.32) -- (80.18,22.02) -- (80.57,20.99) -- (80.96,20.42) -- (81.36,20.38) -- (81.75,20.88) -- (82.15,21.84) -- (82.54,23.09) -- (82.94,24.45) -- (83.33,25.69) -- (83.72,26.62) -- (84.12,27.09) -- (84.51,27.03) -- (84.91,26.46) -- (85.3,25.46) -- (85.7,24.2) -- (86.09,22.88) -- (86.48,21.72) -- (86.88,20.88) -- (87.27,20.51) -- (87.67,20.66) -- (88.06,21.3) -- (88.46,22.33) -- (88.85,23.58) -- (89.24,24.86) -- (89.64,25.96) -- (90.03,26.7) -- (90.43,26.97) -- (90.82,26.74) -- (91.22,26.04) -- (91.61,24.98) -- (92,23.73) -- (92.4,22.5) -- (92.79,21.48) -- (93.19,20.83) -- (93.58,20.65) -- (93.98,20.96) -- (94.37,21.72) -- (94.76,22.8) -- (95.16,24.03) -- (95.55,25.22) -- (95.95,26.16) -- (96.34,26.72) -- (96.74,26.82) -- (97.13,26.42) -- (97.52,25.61) -- (97.92,24.51) -- (98.31,23.3) -- (98.71,22.17) -- (99.1,21.3) -- (99.5,20.83) -- (99.89,20.83) -- (100.29,21.29) -- (100.68,22.15) -- (101.07,23.26) -- (101.47,24.44) -- (101.86,25.52) -- (102.26,26.31) -- (102.65,26.69) -- (103.05,26.61) -- (103.44,26.08) -- (103.83,25.19) -- (104.23,24.07) -- (104.62,22.92) -- (105.02,21.9) -- (105.41,21.19) -- (105.81,20.89) -- (106.2,21.05) -- (106.59,21.64) -- (106.99,22.57) -- (107.38,23.68) -- (107.78,24.81) -- (108.17,25.76) -- (108.57,26.4) -- (108.96,26.61) -- (109.35,26.38) -- (109.75,25.73) -- (110.14,24.77) -- (110.54,23.66) -- (110.93,22.57) -- (111.33,21.68) -- (111.72,21.12) -- (112.11,20.99) -- (112.51,21.3) -- (112.9,22) -- (113.3,22.98) -- (113.69,24.08) -- (114.09,25.13) -- (114.48,25.95) -- (114.87,26.43) -- (115.27,26.49) -- (115.66,26.11) -- (116.06,25.36) -- (116.45,24.37) -- (116.85,23.28) -- (117.24,22.28) -- (117.63,21.52) -- (118.03,21.11) -- (118.42,21.14) -- (118.82,21.58) -- (119.21,22.37) -- (119.61,23.38) -- (120,24.44);
\end{scope}
\end{scope}
\end{scope}
\begin{scope}
\fontspec{PT Sans Caption}
\draw (18.82,57.32) node [text=mycolor6C606B,rotate around={-0: (-2.12,-25.16)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{0.0}$};
\draw (18.82,40.55) node [text=mycolor6C606B,rotate around={-0: (-2.12,-8.39)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{0.5}$};
\draw (18.82,23.77) node [text=mycolor6C606B,rotate around={-0: (-2.12,8.39)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{1.0}$};
\draw (18.82,7) node [text=mycolor6C606B,rotate around={-0: (-2.12,25.16)},left,inner sep=0.0]{\fontsize{2.82mm}{3.39mm}\selectfont $\text{1.5}$};
\end{scope}
\begin{scope}
\fontspec{PT Sans}
\draw (9.29,30.16) node [text=mycolor564A55,draw=mycolor000000,draw opacity=0,rotate around={90: (0,2)},inner sep=0.0]{\fontsize{3.88mm}{4.66mm}\selectfont $\text{$\int_{z=0}^{z=\textbf{r}}J_1(z)dz$}$};
\end{scope}
\end{scope}
\end{tikzpicture}
\end{document}
